### PR TITLE
A Few HTML/CSS changes + prettier.

### PR DESCRIPTION
**Pull request by IlanKro**
A few changes that I worked with Ran and Raphael.
-Changes to font weights.
-Enlarged the Corona img.
-Removed bold from the above the fold text.
-Prettier ran on the CSS so a lot of spacing got changed in the process.
-fixed a type how does it work-> works

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -137,7 +137,7 @@ <h1>
 
     <a class="anchor" id="howItWorks"></a>
     <div class="method-screen">
-      <h1>How does it <b>work?</b></h1>
+      <h1>How does it <b>works?</b></h1>
```
Comment (soryy708) [inline index.html]: That's not correct English. It's either `How does it work` or `How it works`.

File: index.html [hunk 2]
```diff
@@ -194,8 +194,7 @@ <h1>Our <b>Advantages</b></h1>
           <div>
             <p class="title">Marketing and increase in sales</p>
             <p class="description">
-              Advertise to your customers based on data, sales and content while they're waiting in queue.<br />
-              The customer will continue their journey in the business while their place in queue is preserved.
+              Advertise to your customers based on data, sales and content while they're waiting in queue.The customer will continue their journey in the business while their place in queue is preserved.
```
Comment (soryy708) [inline index.html]: Lacks space: `queue.The`

File: new-style.css [hunk 3]
```diff
@@ -1,1140 +1,1144 @@
...
-    font-size: 3.75em;
-    line-height: 1.4166em;
-    margin-bottom: 1em;
-    text-align: center;
+  font-weight: 500;
+  font-size: 3.75em;
+  line-height: 1.4166em;
+  margin-bottom: 1em;
+  text-align: center;
+  font-weight: 400;
```
Comment (soryy708) [inline new-style.css]: If you're adding a `font-weight`, remove the duplicate `font-weight` that is overriden (line 400).

File: index.html (hunk 1 above)
Comment (IlanKro) [reply index.html]: Oh right, "does" cancels present simple, forgot about that rule

Comment (soryy708) [reply index.html]: https://ell.stackexchange.com/questions/17778/how-it-works-vs-how-does-it-work

**Review by soryy708 — APPROVED** [review]

**Pull request merged**